Update active-directory-b2c-ui-customization-custom.md

marcelodiiorio opened the pull request:
Must DataUri be included, in order to avoid the error "The required "DataUri element is missing from ContentDefinition"? Should we comment that we can also add RecoveryUri and Metadata, and that this last one makes necessary to add the Item DisplayName?

azurecla commented:
Hi __@anosov1960__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

marcelodiiorio committed:
Update active-directory-b2c-ui-customization-custom.md

Must <DataUri> be included, in order to avoid the error "The required "DataUri" element is missing from ContentDefinition"? Should we comment that we can also add <RecoveryUri> and Metadata, and that this last one makes necessary to add the Item DisplayName?
